dolly-backend-app/src/main/java/no/nav/dolly/provider/api/OrganisasjonController.java

```diff
@@ -82,6 +85,15 @@ public RsOrganisasjonBestillingStatus hentBestilling(
         return bestillingService.fetchBestillingStatusById(bestillingId);
     }
 
+    @GetMapping("/orgnumre/{orgnumre}")
+    @Cacheable(value = CACHE_ORG_BESTILLING)
+    @Operation(description = "Hent organisasjoner fra forvalter basert på orgnumre")
+    public OrganisasjonDetaljer hentOrganisasjoner(
```

Comment:
Regner med dette er et endepunkt for frontend? Kunne ikke FE ha gått direkte til org-forvalter i stedet?

Reply:
Forøvrig caching er vanskelig å vedlikeholde på en liste, tror ikke det vil virke uten mer konfigurering, fordi default innparametre vil være "liste"-objektet som vil være nytt for hvert kall.

Reply:
Whoops, det ble bare lagt til for å teste at auth virka fra BE uten å sende inn organisasjoner 😅  De blir henta direkte i frontend ja, skal fjerne denne igjen 👍